os-autoinst-obs-auto-submit

```diff
@@ -18,10 +18,33 @@ XMLSTARLET=$(command -v xmlstarlet || true)
 # shellcheck source=/dev/null
 . "$(dirname "${BASH_SOURCE[0]}")"/_common
 
+check_project_exists() {
+    # ex: openSUSE:Backports:SLE-15-SP6
+    local prj="$1"
+    local rc=0
+    if [[ -z "$prj" ]]; then
+        echo "Error: No project provided." >&2
+        return 1
+    fi
+    # dont need the XML output for checking existence.
+    # if $prj does not exist return 404
+    osc meta prj "$prj" > /dev/null 2> /dev/null
```

Comment:
```suggestion
    osc meta prj "$prj" 2>&1 >/dev/null
```

Reply:
That doesn't work like expected. Should be:
```diff
- osc meta prj "$prj" 2>&1 >/dev/null
+ osc meta prj "$prj" >/dev/null 2>&1
```